### PR TITLE
make wellness tips and banner's medium page accessible on settings page

### PR DESCRIPTION
**Pull request by androidmaven**

### CODE REVIEW

File: app/src/main/java/com/hover/stax/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -113,6 +114,13 @@ class SettingsFragment : Fragment(), NavigationInterface {
         }
     }
 
+    private fun setupLearnCard() {
+        with(binding.staxLearn) {
+            learnFinances.setOnClickListener {  findNavController().navigate(R.id.action_navigation_home_to_wellnessFragment) }
```
Comment (alexosugo) [inline app/src/main/java/com/hover/stax/settings/SettingsFragment.kt:119]: The navigation action here is incorrect. The source fragment here is the `settings`.

File: app/src/main/java/com/hover/stax/settings/SettingsFragment.kt [hunk 2]
```diff
@@ -113,6 +114,13 @@ class SettingsFragment : Fragment(), NavigationInterface {
         }
     }
 
+    private fun setupLearnCard() {
```
Comment (alexosugo) [inline app/src/main/java/com/hover/stax/settings/SettingsFragment.kt:117]: The learn card isn't showing up in settings for me.

**Review by alexosugo — CHANGES_REQUESTED** [review]
Navigation to the USSD Lib fragment also appears to be broken.

File: app/src/main/java/com/hover/stax/settings/SettingsFragment.kt (hunk 2 above)
Comment (androidmaven) [reply app/src/main/java/com/hover/stax/settings/SettingsFragment.kt:117]: New push should fix

**Comment by androidmaven** [comment]
Thanks for the correction, @alexosugo

**Pull request merged**